python/arcticdb/version_store/library.py

```diff
@@ -810,6 +810,38 @@ def update(
             prune_previous_version=prune_previous_versions,
         )
 
+    def list_symbols_with_staged_data(self) -> List[str]:
```

Comment:
What's different about this and the other method?

https://github.com/man-group/ArcticDB/blob/master/python/arcticdb/version_store/library.py#L870

Reply:
My comment about 'seeing staged data' in #714, was not about listing symbols, but rather getting the actual inflight appends and updates.  I think that's lower priority.  At least there is a way to cleanup now.

Reply:
No difference, I didn't know it existed. I've removed this method now.